### PR TITLE
11832 compile grpc helloworld

### PR DESCRIPTION
**Pull request by loriadi**

### CODE REVIEW

**Comment by loriadi** [comment]
#build

File: dev/com.ibm.ws.grpc_fat/test-applications/helloworld_client/resources/index.xhtml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE HTML>
```
Comment (wtlucy) [inline dev/com.ibm.ws.grpc_fat/test-applications/helloworld_client/resources/index.xhtml:1]: Can you delete this file?

Comment (loriadi) [reply dev/com.ibm.ws.grpc_fat/test-applications/helloworld_client/resources/index.xhtml]: Done

**Comment by loriadi** [comment]
#run-libby-bot

**Pull request merged**